CompanyEmployees.Core.Domain/Entities/Employee.cs

```diff
@@ -20,7 +20,10 @@ public class Employee
         public string? Position { get; set; }
 
         [ForeignKey(nameof(Company))]
```

Comment:
Remove this temporary `salary` field before merging. It is redundant with the `Salary` property, uses inconsistent casing, and violates clean architecture principles by leaving non-functional code in the domain model.

```suggestion
// Removed temporary field before merging
```